### DISCUSSION TITLE
Is Hyper dying?

### DISCUSSION THREAD

**Discussion by alimxk** [vercel/hyper#8034] [General]
The latest tag `v3.4.1` was released on Jan 7, 2023
Pretty much most of the PRs since were that of dependabot bumping dependencies
There are 900+ issues open
Are there any plans to maintain this project going forward?

**Comment by chrowe** [comment]
Based on https://github.com/vercel/hyper/discussions/7770 I would say yes

**Comment by jacksonSingleton** [comment]
Wezterm and now ghostty are the trendy terms so hyper will likely fall to the wayside. May be time for a community fork, but as far as terminal performance goes, hyper just isn't there.
